store/storetest/job_store.go

```diff
@@ -226,15 +205,11 @@ func testJobGetAllByStatus(t *testing.T, ss store.Store) {
 		defer ss.Job().Delete(job.Id)
 	}
 
-	if received, err := ss.Job().GetAllByStatus(status); err != nil {
-		t.Fatal(err)
-	} else if len(received) != 3 {
-		t.Fatal("received wrong number of jobs")
-	} else if received[0].Id != jobs[1].Id || received[1].Id != jobs[0].Id || received[2].Id != jobs[2].Id {
-		t.Fatal("should've received jobs ordered by CreateAt time")
-	} else if received[1].Data["test"] != "data" {
-		t.Fatal("should've received job data field back as saved")
-	}
+	received, err := ss.Job().GetAllByStatus(status)
+	require.Nil(t, err)
+	require.Equal(t, 3, len(received), "received wrong number of jobs")
+	require.False(t, received[0].Id != jobs[1].Id || received[1].Id != jobs[0].Id || received[2].Id != jobs[2].Id, "should've received jobs ordered by CreateAt time")
```

Comment:
Here you can use three lines using different `require.Equal` for each or clause.